test/config-next/sfe.json

```diff
@@ -1,7 +1,6 @@
 {
 	"sfe": {
 		"listenAddress": "0.0.0.0:4003",
-		"debugAddr": ":8015",
```

Comment:
I feel like the SFE should probably retain its debugAddr -- it's a long-lived process that we want to be able to get metrics from.

More generally, I don't think we should remove debugAddr from any of our long-lived commands until and unless we remove it from _all_ of them (to be wholly replaced by the CLI flag).

Reply:
The SFE retains its support for debugAddr in this PR, and that's still tested in test/config/sfe.json[^1]. This diff removes the configuration of debugAddr in `config-next`, to match the other services in config-next (and prod).

[^1]: Well, not tested, exactly, because we do pass `--debug-addr` in test/startservers.py regardless of config vs config-next.